kafka-ui-react-app/src/components/common/NewTable/SizeCell.tsx

```diff
@@ -3,8 +3,15 @@ import { CellContext } from '@tanstack/react-table';
 import BytesFormatted from 'components/common/BytesFormatted/BytesFormatted';
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-const SizeCell: React.FC<CellContext<any, unknown>> = ({ getValue }) => (
-  <BytesFormatted value={getValue<string | number>()} />
+type AsAny = any;
```

Comment:
why are we added this ?

Reply:
After changes of the `SizeCell` props the props string needs to be formatted as below and the eslint-disable comment ignored, so I added `AsAny`, to ignore it before props declaration